conference_table_api.py

```
@@ -0,0 +1,108 @@
+        messages.append(message)
+
+    return jsonify({
+        "status": "success",
+        "message_id": message['id'],
+        "total_messages": len(messages)
+    })
+
+@app.route('/messages', methods=['GET'])
+def get_messages():
+    """Get all messages"""
+    agent_id = request.args.get('agent_id')
```

Ruff (0.14.3)

Local variable `agent_id` is assigned to but never used

Remove assignment to unused variable `agent_id`

(F841)